bespin_k8s_gcloud_cluster/tasks/create_k8s_cluster.yml

```diff
@@ -0,0 +1,54 @@
+- name: "Creating k8s cluster {{ cluster_name }} in zone {{ gcloud_compute_zone }}"
+  command: >
+    gcloud container clusters create "{{ cluster_name }}"
+    --zone "{{ gcloud_compute_zone }}"
+    --machine-type "{{ gcloud_machine_type }}"
+    --enable-autoscaling
+    --no-enable-autoupgrade
+    --num-nodes "{{ gcloud_num_nodes }}"
+    --min-nodes "{{ gcloud_min_nodes }}"
+    --max-nodes "{{ gcloud_max_nodes }}"
+  when: not cluster_exists
+
+- name: Create the large-machine-pool node pool
+  command: >
+    gcloud container node-pools create large-machine-pool
+    --cluster {{ cluster_name }}
+    --zone {{ gcloud_compute_zone }}
+    --enable-autoscaling
+    --machine-type {{ large_pool_machine_type }}
+    --num-nodes {{ large_pool_num_nodes }}
+    --min-nodes {{ large_pool_min_nodes }}
+    --max-nodes {{ large_pool_max_nodes }}
+  when: not large_node_pool_exists
+
+- name: Configure kubectl command line access to the cluster
+  command: >
+    gcloud container clusters get-credentials
+    "{{ cluster_name }}"
+    "--zone={{ gcloud_compute_zone }}"
+
+- name: "Check if bespin-deploy service account exists"
```

Comment:
This and the following steps in `create_k8s_cluster.yml` are setting up the service account, role binding, and extracting the token/cert to be used by the `k8s` module.

Reply:
I see, the next steps use different credentials, but I'm not sure that explains why this step can't use the `k8s` module. Am I missing something?

Reply:
The `k8s` module can't directly use the kube config produced by `gcloud container clusters get-credentials`. That kubeconfig doesn't have a `access-token` which is required by the `k8s` ansible module. Instead of all this extra setup in the latest version I create a temporary kbeconfig and run `kubectl` once to fill in the `access-token`.